guides/common/modules/proc_enabling-the-installer-managed-dns-service.adoc

```diff
@@ -0,0 +1,52 @@
+[id="enabling-the-installer-managed-dns-service_{context}"]
+= Enabling the installer-managed DNS service
+
+If you do not have a DNS server available in your network, you can use the installer-managed DNS service. This feature enables you to provide a DNS service with a low maintenance effort.
+
+You can use a backup of the answer file that was created before configuring external DNS, or you can create a backup of the answer file.
+ifndef::orcharhino[]
+For more information about answer files, see {InstallingServerDocURL}configuring-server_{project-context}[Configuring {ProjectServer}].
+endif::[]
```

Comment:
I'd drop this so it's no longer "you go from external DNS to internal" but really just "this is how you enable the installer managed service.
```suggestion
```

Reply:
removed